aws/resource_aws_ebs_volume.go

```diff
@@ -46,7 +46,8 @@ func resourceAwsEbsVolume() *schema.Resource {
 				Optional: true,
 				Computed: true,
 				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
-					return d.Get("type").(string) != ec2.VolumeTypeIo1 && new == "0"
+					return (d.Get("type").(string) != ec2.VolumeTypeIo1 && new == "0") ||
+					    (d.Get("type").(string) != ec2.VolumeTypeIo2 && new == "0")
```

Comment:
This code could be simplified a little but this should at least fix the current lint failure 👍 

```suggestion
					return (d.Get("type").(string) != ec2.VolumeTypeIo1 && new == "0") || (d.Get("type").(string) != ec2.VolumeTypeIo2 && new == "0")
```

Reply:
it worked, thanks